src/main/scala/firrtl_interpreter/LoFirrtlExpressionEvaluator.scala

```diff
@@ -391,7 +391,8 @@ class LoFirrtlExpressionEvaluator(val dependencyGraph: DependencyGraph, val circ
         case c: SIntLiteral => Concrete(c).forceWidth(c.tpe)
         case blackBoxOutput: BlackBoxOutput =>
           log(s"got a black box, $blackBoxOutput")
-          val concreteInputs = blackBoxOutput.dependentInputs.map { input => getValue(input)}
+          val concreteInputs: immutable.Seq[Concrete] =
```

Comment:
I think it's better to leave this as `Seq` if you can, that is `immutable.Seq` in Scala 2.13, but `collection.Seq` in 2.12.

Reply:
Also re-worked